homeassistant/components/reolink/entity.py

```diff
@@ -42,7 +50,11 @@ class ReolinkHostCoordinatorEntity(CoordinatorEntity[DataUpdateCoordinator[None]
     """
 
     _attr_has_entity_name = True
-    entity_description: ReolinkHostEntityDescription | ReolinkChannelEntityDescription
+    entity_description: (
+        ReolinkHostEntityDescription
+        | ReolinkChannelEntityDescription
+        | ReolinkChimeEntityDescription
+    )
```

Comment:
On another note, why do we overwrite all of these types? I assume that we only use the `cmd_key` here, so Ideally we could create a ReolinkEntityDescription which adds that and extend that (and overwrite where needed)

But for a followup I think

Reply:
Because the `supported` uses diffrent types:
```
supported: Callable[[Host, int], bool] = lambda api, ch: True
supported: Callable[[Host], bool] = lambda api: True
supported: Callable[[Chime], bool] = lambda chime: True
```

Reply:
Yes, but we can extend it where we needed. You can create 1 generic entity description and have the rest extend from that

Reply:
I know, but 95% of the Reolink entities are using the `supported` key, so I would need to extend it everywhere....

Reply:
you only need to extend it 3 times?